Tempura/Sources/Core/ViewControllerModellableView.swift

```diff
@@ -70,46 +70,8 @@ public extension ViewControllerModellableView {
 /// see also https://developer.apple.com/documentation/uikit/uiview/positioning_content_relative_to_the_safe_area
 public extension UIView {
   /// Implementation of safeAreaInsets in order to be accessible even to older iOS versions.
+  @available(*, deprecated, message: "Use `self.safeAreaInsets` instead")
   var universalSafeAreaInsets: UIEdgeInsets {
```

Comment:
why not removing altogether? It's going to be a breaking release regardless, and it shouldn't be more than a search&replace changes for the apps